operators/cmd/tenant-operator/main.go

```diff
@@ -88,6 +91,9 @@ func main() {
 	flag.StringVar(&ncURL, "nc-url", "", "The base URL for the nextcloud actor.")
 	flag.StringVar(&ncTnOpUser, "nc-tenant-operator-user", "", "The username of the acting account for nextcloud.")
 	flag.StringVar(&ncTnOpPsw, "nc-tenant-operator-psw", "", "The password of the acting account for nextcloud.")
+	flag.DurationVar(&requeueTimeMinimum, "nc-tenant-operator-rq-time-min", 4*time.Hour, "Minimum time elapsed before requeue of controller.")                                                                                                                                              // 30*time.Second for testing
+	flag.DurationVar(&requeueTimeMaximum, "nc-tenant-operator-rq-time-max", 8*time.Hour, "Maximum time elapsed before requeue of controller.")                                                                                                                                              // 35*time.Second for testing
+	flag.DurationVar(&tenantWorkspaceKeepAlive, "nc-tenant-operator-wspc-keep-alive", time.Hour, "Time elapsed after last login of tenant during which the tenant workspace should be kept alive: after this period, the controller will attempt to delete the tenant personal workspace.") // 80**time.Second for testing
```

Comment:
```suggestion
	flag.DurationVar(&requeueTimeMinimum, "tenant-operator-rq-time-min", 4*time.Hour, "Minimum time elapsed before requeue of controller.")                                                                                                                                              // 30*time.Second for testing
	flag.DurationVar(&requeueTimeMaximum, "tenant-operator-rq-time-max", 8*time.Hour, "Maximum time elapsed before requeue of controller.")                                                                                                                                              // 35*time.Second for testing
	flag.DurationVar(&tenantNSKeepAlive, "tenant-operator-ns-keep-alive", time.Hour, "Time elapsed after last login of tenant during which the tenant namespace should be kept alive: after this period, the controller will attempt to delete the tenant personal namespace.") // 80**time.Second for testing
```

Reply:
I would also ask to drop the comments.

Reply:
Resolved in [1bc5a0b](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/1bc5a0bc59369963f844df3167306f056e728c06)